src/Controller/Api/PersonController.php

```diff
@@ -483,7 +484,11 @@ public function deleteAction(int $id)
      */
     public function getPerson(Person $person): Response
     {
-        return $this->makeJsonResponse([
+        $posix = false;
+
+        $currentAccount = $this->getUser();
```

Comment:
I would like to see this close to where it's used. Like on line 509 or so.

Reply:
not sure why github not showing this is outdated, but I did move it.